_layouts/workshop.html

```diff
@@ -9,10 +9,26 @@
   <head>
     <title>Software Carpentry: {{ page.venue }}</title>
     {% include header.html %}
-    <meta name="venue" content="{{page.venue}}" />
-    <meta name="date" content="{{page.humandate}}" />
+    <meta name="slug" content="{{site.github.project_title}}" />
     <meta name="startdate" content="{{page.startdate}}" />
     <meta name="enddate" content="{{page.enddate}}" />
+    <meta name="country" content="{{page.country}}" />
+    <meta name="venue" content="{{page.venue}}" />
+    <meta name="address" content="{{page.address}}" />
+    <meta name="latlng" content="{{page.latlng}}" />
+    <meta name="language" content="{{page.language}}" />
+    <meta name="eventbrite" content="{{page.eventbrite}}" />
+    {% if page.instructor == "FIXME" %}
+    <meta name="instructor" content="{{page.instructor}}" />
+    {% else %}
+    <meta name="instructor" content="{{page.instructor|join:','}}" />
```

Comment:
I'd rather join instructors with pipes (`|`), so we don't have to worry about folks with commas in their names (e.g. “John Smith, Jr.”).  I haven't heard of anyone with a pipe in their name, so I think we should be safe with that delimiter ;).